docs/contributing/marketer.rst

```diff
@@ -39,15 +37,13 @@ If you need help, please reach out to the team. When you start working on a task
 Work in the public domain
 *************************
 
-Unless there is significant reason not to, we default to being open and transparent. We work in the open, usually on our shared :xref:`Mautic Google Drive` or by sharing our work with the team on our Canva account. This ensures that if - for whatever reason - someone is unable to complete a task, it is easy for another contributor to pick up where they left off. It also means we can always find previous work that had been done if it needs to be re-used in the future.
+Unless there's significant reason not to, the default approach is openness and transparency. Work happens in the open, usually on the shared :xref:`Mautic Google Drive` or through the team Canva account. This ensures that if someone is unable to complete a task, it's easy for another contributor to pick up where they left off. It also means contributors can always find previous work if needed for future re-use. 
 
-Please always ensure that you upload your work at regular - ideally daily - intervals. You can use the file prefix of WIP-filename to indicate that it is currently in progress.
+Please always ensure that you upload work at regular—ideally daily—intervals. Use the file prefix of WIP-filename to indicate that it's currently in progress.
 
 Update regularly
 ****************
 
 Please provide regular updates on the issue. If you can’t complete the task, add a comment on the issue or let your Team Lead know so someone else can take it over.
 
-We totally understand that life happens and it's easy to take on too much. No judgement at all. We try to be respectful of each other by ensuring we give as much notice as possible if we're not going to be able to fulfil a task assigned to us.
-
-.. vale on
+The team understands that life happens and it's easy to take on too much. No judgment. Contributors respect each other by providing as much notice as possible when they're unable to fulfill an assigned task.
```

Comment:
I think we can adjust this a bit to get as closer as possible to the original version.

```suggestion
Life can be unpredictable, and it’s easy to feel overwhelmed by trying to manage too much at once. There's no judgment here. The team's goal is to respect one another by giving as much notice as possible if someone can't complete an assigned task.
```